assets/model_monitoring/components/src/action_analyzer/action_analyzer_correlation_test/run.py

```diff
@@ -144,6 +147,7 @@ def run():
     print(group_set)
     print("===All indexes===")
     print(index_set)
+    data_with_action_metric_score_df = data_with_action_metric_score_df.filter(col(INDEX_SCORE_LLM_COLUMN) > DEFAULT_RETRIEVAL_SCORE)  # noqa: E501
```

Comment:
This is to filter out the failed one? e.g. we get -1 if the LLM call failed?

Reply:
set as 0 if failed.